src/model/AssetsCompiler.ts

```diff
@@ -107,10 +109,25 @@ export class AssetsCompiler {
         private readonly runConfig: IAssetsCompilerConfiguration,
         private readonly repositoryDir: string
     ) {
+        if (AssetsCompiler.shouldUseAscLocal()) {
+            console.warn(
+                cwarn`Starting from the cplace release 23.2 you should be using @cplace/asc-local instead of @cplace/asc!`
+            );
+        }
         this.repositoryName = path.basename(repositoryDir);
         this.projects = this.setupProjects();
     }
 
+    private static shouldUseAscLocal(): boolean {
+        const version = CplaceVersion.get();
+        const isMajorHigherThen23 = version.major > 23;
+        const isMinorHigherOrEqual2 =
+            version.major === 23 && version.minor >= 2;
+        const is23_1Snapshot =
```

Comment:
Why support the snapshot? This might qualify for a comment on why this is here.

Reply:
Because when we merge node 18 stuff, all this will be in the current master (which is 23.1 - Snapshot), and we need to use @cplace/asc-local from that moment